eng/Versions.props

```diff
@@ -204,7 +204,7 @@
     <!-- Packages from 2.1/2.2 branches used for site extension build -->
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>2.1.1</MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>2.2.0</MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>
-    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.3-servicing.20163.14</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
+    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.3-servicing-20113-19</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
```

Comment:
Going back in time?

Reply:
I'm not sure where these versions are coming from, was going to see any of the build folks knew.

Reply:
Could you give me an example of a warning related to using the 3.1.3-servicing.20163.14 version?

Reply:
```
##[warning]src\SiteExtensions\LoggingAggregate\src\Microsoft.AspNetCore.AzureAppServices.SiteExtension\Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj(0,0): warning NU1603: Microsoft.AspNetCore.AzureAppServices.SiteExtension depends on Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x64 (>= 3.1.3-servicing.20163.14) but Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x64 3.1.3-servicing.20163.14 was not found. An approximate best match of Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x64 3.1.3-servicing-20113-19 was resolved.
```
https://dev.azure.com/dnceng/internal/_build/results?buildId=610167&view=logs&j=1b89928a-2219-5ef9-602f-f95beb3da4dc&t=39263dad-2be6-5f40-bde4-1bab0d7931b4&l=862

Reply:
Looks more like a feed we need to build correctly has been removed from NuGet.config. @wtgodbe❔

Reply:
Just double-checked. Undo changes to this file and instead add missing feed to NuGet.config:
``` xml
 <add key="dotnet31-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" />
```
Feel free to include a comment so we don't forget 😸

Reply:
Thanks 👍